libmattermost.c

```diff
@@ -3526,6 +3584,9 @@ mm_add_account_options(GList *account_options)
 	
 	option = purple_account_option_bool_new(N_("Use SSL/HTTPS"), "use-ssl", TRUE);
 	account_options = g_list_append(account_options, option);
+
+	option = purple_account_option_bool_new(N_("Use password as MMAUTHTOKEN"), "use-mmauthtoken", FALSE);

```

Comment:
Rather than a technical description of the problem being solved, could we refer to what we're trying to do (especially for non-native English readers)

Suggestions:
* Use GitLab cookie for authentication
* Password is GitLab cookie
* GitLab authentication workaround
* something else?